project_transport_only/dimensions/supplemental/simplified_vehicle_types.csv

```diff
@@ -0,0 +1,15 @@
+id,name
+bev_compact,BEV Compact Car
+bev_midsize,BEV Midsize Car
+bev_pickup,BEV Pickup
+bev_suv,BEV SUV
+phev_compact,PHEV Compact Car
+phev_midsize,PHEV Midsize Car
+phev_pickup,PHEV Pickup
+phev_suv,PHEV SUV
+bev_light-medium_freight_truck,BEV Light-Medium Freight Truck
+bev_medium_freight_truck,BEV Medium Freight Truck
+bev_vocational_truck,BEV Non-Freight Truck
```

Comment:
```suggestion
bev_vocational_truck,BEV Vocational Truck
```

I think you use "BEV Vocational Truck" as the name under project? Maybe just copy-paste all those lines to be sure they're the same in both places?